src/js/tabs/osd.js

```diff
@@ -972,6 +972,14 @@ OSD.constants = {
             positionable: true,
             preview: [ "22600", "22600", "22600", "22600"]
         },
+        OSD_PROFILE_NAME: {
+            name: 'OSD_PROFILE_NAME',
+            desc: 'osdDescElementOsdProfileName',
```

Comment:
You need to add this key to the English `messages.json` file.

Reply:
@McGiverGim Thanks, just did it now.  This is my first Configurator change so it was pretty much monkey see-monkey do... :o)